app/views/jobseekers/job_applications/build/_caption.html.slim

```diff
@@ -0,0 +1 @@
+.govuk-caption-m = t("jobseekers.job_applications.current_step", current: JobApplication.completed_steps[step] + 1, total: JobApplication.completed_steps.values.last + 2)
```

Comment:
I don't think this is your fault but while we're here, it's really weird that our captions say "Step X of 10" but the steps component on the right only goes up to 9. (It doesn't include the 'review' step.)

Reply:
Aha - Figma says the error is in the steps component, not in the caption.

Reply:
I'll make a little ticket to address that

Reply:
Slash I'll ask if Joe wants to do it

Reply:
Haha I missed that on Joe's PR, slack him and see if he wants to just update it in the morning